documentation/book/con-custom-resources-example.adoc

```diff
@@ -50,13 +52,15 @@ spec: <2>
 <2> The specification for this CRD, including the group (domain) name, the plural name and the supported schema version, which are used in the URL to access the API of the topic. The other names are used to identify instance resources in the CLI. For example, `kubectl get kafkatopic my-topic` or `kubectl get kafkatopics`.
 <3> The shortname can be used in CLI commands. For example, `kubectl get kt` can be used as an abbreviation instead of `kubectl get kafkatopic`.
 <4> The information presented when using a `get` command on the custom resource.
-<5> openAPIV3Schema validation provides validation for the creation of topic custom resources. For example, a topic requires at least one partition and one replica.
+<5> The current status of the CRD.
+<6> openAPIV3Schema validation provides validation for the creation of topic custom resources. For example, a topic requires at least one partition and one replica.
```

Comment:
and constraints?

Reply:
I'm not aware of any constraints. I'll clarify that the information available is "as described in the schema reference for the resource".